docs/API-Reference/bpmn-js/importXML.md

```diff
@@ -10,3 +10,62 @@ grand_parent: "API Reference"
 ---
 
 # importXML
+
+`importXML` API parses and renders a BPMN 2.0 diagram.
+
+## Usage
+
+importXML(xml: _[String](#)_, bpmnDiagram: _[ModdleElement](#)_ or _[String](#)_): Promise<_[ImportXMLResult](#importxmlresult)_, _[ImportXMLError](#importxmlerror)_>
+
+* `xml`: The BPMN 2.0 XML.
+* `bpmnDiagram`: The BPMN diagram or the id of the diagram to render _(if not provided, the first one will be rendered)_.
+
+### Code Example
+
+```javascript
+const modeler = new BpmnJS();
```

Comment:
The import statement is missing in the example.

Reply:
You don't need an import statement every time, e.g. when you integrate the production builds directly inside the browser.

Reply:
I removed the BpmnJS construction code and added a comment that says "modeler is an instance of BpmnJS" instead since the BpmnJS constructor also expects the container DOM element. So it does not make sense to include those in every API Reference code snippets. Let's make the code snippets focus on only the API.